packages/interbit/src/tests/testData/index.js

```diff
@@ -220,13 +225,24 @@ const defaultManifest = {
   },
   manifest: {
     interbitRoot: {
+      alias: 'interbitRoot',
       chainId:
         '56399770d33b17fe3ba20a1d911cc56c3cc47ed11fc8f6935543c2b25da96f47',
+      chainIdMap: {
+        control:
+          '7988dc0fe96bf5371909781b9cab0de21fcca8a740573267b1134e6889922162',
+        public:
+          '28884c86bb5f73e48726ebdc466d45f4034a22c7ce9b69e5b75bd94d55f78d7f'
+      },
       validators: ['SuperSecurePubKey'],
       covenant: 'interbitRoot',
-      covenants: {
-        interbitRoot: 'interbitRoot',
-        control: 'control'
+      covenantHashMap: {
+        interbitRoot:
+          '4fd5sca61c3af35ecca31e58c33c368e1554dc1dab01ab8fc3aadc965f608775',
+        control:
+          '1abaa91370f606b575ba2bca4dc13a3717aba7c6b09b95b23334c89eb98650fa',
+        public:
+          '72fe3ca61c3af35ecca31e58c33c368e1554dc1dab01ab8fc3aadc965f608775'
```

Comment:
`chainIdMap` and `covenantHashMap` were good name choices, especially since both are hashes and there are duplicate aliases.

Reply:
Thanks!